### ISSUE TITLE
Error when using Stripes Payment Forms field type

### ISSUE THREAD

**Issue by iamkeir** [enupal/stripe#58]
### Description

When trying to select a payment form from the Stripe Payment Forms field type, receive Craft error: `Unknown scenario: live`

### Steps to reproduce

1. Create a field that uses the Stripes Payment Forms field type
2. Attach field to a section
3. Choose a payment form
4. Receive error on save

Video demonstration: https://jmp.sh/FpwIMq0

### Temporary workaround

Use a plain text field and insert Payment Form handle manually.

### Additional info

- Craft version: Craft Pro 3.3.15
- PHP version: 7.3.1
- Database driver & version: MySQL 5.7.25
- Plugin version: 2.1.3

### Full error report

```
yii\base\InvalidArgumentException: Unknown scenario: live in /Users/iamkeir/Sites/ben/build/vendor/yiisoft/yii2/base/Model.php:357
Stack trace:
#0 /Users/iamkeir/Sites/ben/build/vendor/craftcms/cms/src/fields/BaseRelationField.php(335): yii\base\Model->validate()
#1 /Users/iamkeir/Sites/ben/build/vendor/craftcms/cms/src/fields/BaseRelationField.php(284): craft\fields\BaseRelationField::_validateRelatedElement(Object(enupal\stripe\elements\PaymentForm))
#2 /Users/iamkeir/Sites/ben/build/vendor/craftcms/cms/src/base/Element.php(1255): craft\fields\BaseRelationField->validateRelatedElements(Object(craft\elements\MatrixBlock), NULL)
#3 [internal function]: craft\base\Element->validateCustomFieldAttribute('field:membershi...', Array, Object(yii\validators\InlineValidator))
#4 /Users/iamkeir/Sites/ben/build/vendor/yiisoft/yii2/validators/InlineValidator.php(72): call_user_func(Array, 'field:membershi...', Array, Object(yii\validators\InlineValidator))
#5 /Users/iamkeir/Sites/ben/build/vendor/yiisoft/yii2/validators/Validator.php(257): yii\validators\InlineValidator->validateAttribute(Object(craft\elements\MatrixBlock), 'field:membershi...')
#6 /Users/iamkeir/Sites/ben/build/vendor/yiisoft/yii2/base/Model.php(367): yii\validators\Validator->validateAttributes(Object(craft\elements\MatrixBlock), Array)
#7 /Users/iamkeir/Sites/ben/build/vendor/craftcms/cms/src/fields/Matrix.php(686): yii\base\Model->validate()
#8 /Users/iamkeir/Sites/ben/build/vendor/craftcms/cms/src/base/Element.php(1255): craft\fields\Matrix->validateBlocks(Object(craft\elements\GlobalSet), NULL)
#9 [internal function]: craft\base\Element->validateCustomFieldAttribute('field:membershi...', Array, Object(yii\validators\InlineValidator))
#10 /Users/iamkeir/Sites/ben/build/vendor/yiisoft/yii2/validators/InlineValidator.php(72): call_user_func(Array, 'field:membershi...', Array, Object(yii\validators\InlineValidator))
#11 /Users/iamkeir/Sites/ben/build/vendor/yiisoft/yii2/validators/Validator.php(257): yii\validators\InlineValidator->validateAttribute(Object(craft\elements\GlobalSet), 'field:membershi...')
#12 /Users/iamkeir/Sites/ben/build/vendor/yiisoft/yii2/base/Model.php(367): yii\validators\Validator->validateAttributes(Object(craft\elements\GlobalSet), Array)
#13 /Users/iamkeir/Sites/ben/build/vendor/craftcms/cms/src/services/Elements.php(1914): yii\base\Model->validate()
#14 /Users/iamkeir/Sites/ben/build/vendor/craftcms/cms/src/services/Elements.php(525): craft\services\Elements->_saveElementInternal(Object(craft\elements\GlobalSet), true, false)
#15 /Users/iamkeir/Sites/ben/build/vendor/craftcms/cms/src/controllers/GlobalsController.php(256): craft\services\Elements->saveElement(Object(craft\elements\GlobalSet))
#16 [internal function]: craft\controllers\GlobalsController->actionSaveContent()
#17 /Users/iamkeir/Sites/ben/build/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#18 /Users/iamkeir/Sites/ben/build/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#19 /Users/iamkeir/Sites/ben/build/vendor/craftcms/cms/src/web/Controller.php(187): yii\base\Controller->runAction('save-content', Array)
#20 /Users/iamkeir/Sites/ben/build/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('save-content', Array)
#21 /Users/iamkeir/Sites/ben/build/vendor/craftcms/cms/src/web/Application.php(299): yii\base\Module->runAction('globals/save-co...', Array)
#22 /Users/iamkeir/Sites/ben/build/vendor/craftcms/cms/src/web/Application.php(565): craft\web\Application->runAction('globals/save-co...', Array)
#23 /Users/iamkeir/Sites/ben/build/vendor/craftcms/cms/src/web/Application.php(278): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#24 /Users/iamkeir/Sites/ben/build/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#25 /Users/iamkeir/Sites/ben/build/web/index.php(21): yii\base\Application->run()
#26 {main}
```

**Comment by iamkeir** [comment]
Update - if you leave `Validate related elements` unchecked in the field setup, it does not throw the error.

**Comment by iamkeir** [comment]
Further update - when retrieving the Payment Form field data, everything appears to be null...
```
object(enupal\stripe\elements\db\PaymentFormsQuery)#1700 (79) {
  ["id"]=>
  NULL
  ["dateCreated"]=>
  NULL
  ["name"]=>
  NULL
  ["handle"]=>
  NULL
  ["enableCheckout"]=>
  NULL
  ["paymentType"]=>
  NULL
  ["elementType"]=>
  string(34) "enupal\stripe\elements\PaymentForm"
  ["query"]=>
  NULL
  ["subQuery"]=>
  NULL
  ["contentTable"]=>
  string(12) "{{%content}}"
  ["customFields"]=>
  NULL
  ["inReverse"]=>
  bool(false)
  ["asArray"]=>
  bool(false)
  ["ignorePlaceholders"]=>
  bool(false)
  ["drafts"]=>
  bool(false)
  ["draftId"]=>
  NULL
  ["draftOf"]=>
  NULL
  ["draftCreator"]=>
  NULL
  ["revisions"]=>
  bool(false)
  ["revisionId"]=>
  NULL
  ["revisionOf"]=>
  NULL
  ["revisionCreator"]=>
  NULL
  ["uid"]=>
  NULL
  ["fixedOrder"]=>
  bool(false)
  ["status"]=>
  array(1) {
    [0]=>
    string(7) "enabled"
  }
  ["archived"]=>
  bool(false)
  ["trashed"]=>
  bool(false)
  ["dateUpdated"]=>
  NULL
  ["siteId"]=>
  int(1)
  ["unique"]=>
  bool(false)
  ["preferSites"]=>
  bool(false)
  ["enabledForSite"]=>
  bool(true)
  ["leaves"]=>
  bool(false)
  ["relatedTo"]=>
  NULL
  ["title"]=>
  NULL
  ["slug"]=>
  NULL
  ["uri"]=>
  NULL
  ["search"]=>
  NULL
  ["ref"]=>
  NULL
  ["with"]=>
  NULL
  ["orderBy"]=>
  array(1) {
    ["relations.sortOrder"]=>
    int(4)
  }
  ["withStructure"]=>
  NULL
  ["structureId"]=>
  NULL
  ["level"]=>
  NULL
  ["hasDescendants"]=>
  NULL
  ["ancestorOf"]=>
  NULL
  ["ancestorDist"]=>
  NULL
  ["descendantOf"]=>
  NULL
  ["descendantDist"]=>
  NULL
  ["siblingOf"]=>
  NULL
  ["prevSiblingOf"]=>
  NULL
  ["nextSiblingOf"]=>
  NULL
  ["positionedBefore"]=>
  NULL
  ["positionedAfter"]=>
  NULL
  ["defaultOrderBy":protected]=>
  array(1) {
    ["elements.dateCreated"]=>
    int(3)
  }
  ["_placeholderCondition":"craft\elements\db\ElementQuery":private]=>
  NULL
  ["_placeholderSiteIds":"craft\elements\db\ElementQuery":private]=>
  NULL
  ["_result":"craft\elements\db\ElementQuery":private]=>
  NULL
  ["_resultCriteria":"craft\elements\db\ElementQuery":private]=>
  NULL
  ["_searchScores":"craft\elements\db\ElementQuery":private]=>
  NULL
  ["select"]=>
  array(1) {
    ["**"]=>
    string(2) "**"
  }
  ["selectOption"]=>
  NULL
  ["distinct"]=>
  NULL
  ["from"]=>
  NULL
  ["groupBy"]=>
  NULL
  ["join"]=>
  array(1) {
    [0]=>
    array(3) {
      [0]=>
      string(10) "INNER JOIN"
      [1]=>
      string(24) "{{%relations}} relations"
      [2]=>
      array(4) {
        [0]=>
        string(3) "and"
        [1]=>
        string(40) "[[relations.targetId]] = [[elements.id]]"
        [2]=>
        array(2) {
          ["relations.sourceId"]=>
          string(4) "4087"
          ["relations.fieldId"]=>
          string(3) "150"
        }
        [3]=>
        array(3) {
          [0]=>
          string(2) "or"
          [1]=>
          array(1) {
            ["relations.sourceSiteId"]=>
            NULL
          }
          [2]=>
          array(1) {
            ["relations.sourceSiteId"]=>
            string(1) "1"
          }
        }
      }
    }
  }
  ["having"]=>
  NULL
  ["union"]=>
  NULL
  ["params"]=>
  array(0) {
  }
  ["queryCacheDuration"]=>
  NULL
  ["queryCacheDependency"]=>
  NULL
  ["_events":"yii\base\Component":private]=>
  array(0) {
  }
  ["_eventWildcards":"yii\base\Component":private]=>
  array(0) {
  }
  ["_behaviors":"yii\base\Component":private]=>
  array(1) {
    ["customFields"]=>
    object(craft\behaviors\ElementQueryBehavior)#1693 (125) {
      ["seoImage"]=>
      NULL
      ["seoTitle"]=>
      NULL
      ["seoDescription"]=>
      NULL
      ["seoKeywords"]=>
      NULL
      ["logos"]=>
      NULL
      ["logoLink"]=>
      NULL
      ["logoImage"]=>
      NULL
      ["logoTitle"]=>
      NULL
      ["entryTitle"]=>
      NULL
      ["introTitle"]=>
      NULL
      ["introCopy"]=>
      NULL
      ["email"]=>
      NULL
      ["entryCopy"]=>
      NULL
      ["website"]=>
      NULL
      ["phone"]=>
      NULL
      ["role"]=>
      NULL
      ["cv"]=>
      NULL
      ["image"]=>
      NULL
      ["instagram"]=>
      NULL
      ["twitter"]=>
      NULL
      ["linkedIn"]=>
      NULL
      ["imdb"]=>
      NULL
      ["youTube"]=>
      NULL
      ["vimeo"]=>
      NULL
      ["bio"]=>
      NULL
      ["credits"]=>
      NULL
      ["creditProducer"]=>
      NULL
      ["creditTitle"]=>
      NULL
      ["creditLength"]=>
      NULL
      ["creditGenre"]=>
      NULL
      ["showreel"]=>
      NULL
      ["pageTitle"]=>
      NULL
      ["jobStartDate"]=>
      NULL
      ["jobEndDate"]=>
      NULL
      ["jobContactRole"]=>
      NULL
      ["jobLocation"]=>
      NULL
      ["jobEquipmentRequired"]=>
      NULL
      ["postcode"]=>
      NULL
      ["company"]=>
      NULL
      ["marketing"]=>
      NULL
      ["jobContactName"]=>
      NULL
      ["jobContactCompany"]=>
      NULL
      ["jobContactWebsite"]=>
      NULL
      ["jobContactPhone"]=>
      NULL
      ["jobContactEmail"]=>
      NULL
      ["equipmentUsed"]=>
      NULL
      ["genres"]=>
      NULL
      ["jobStatus"]=>
      NULL
      ["skills"]=>
      NULL
      ["newsCopy"]=>
      NULL
      ["credit1Title"]=>
      NULL
      ["credit1Length"]=>
      NULL
      ["credit1Genre"]=>
      NULL
      ["credit1Producer"]=>
      NULL
      ["credit2Title"]=>
      NULL
      ["credit2Length"]=>
      NULL
      ["credit2Genre"]=>
      NULL
      ["credit2Producer"]=>
      NULL
      ["credit3Title"]=>
      NULL
      ["credit3Length"]=>
      NULL
      ["credit3Genre"]=>
      NULL
      ["credit3Producer"]=>
      NULL
      ["credit4Title"]=>
      NULL
      ["credit4Length"]=>
      NULL
      ["credit4Genre"]=>
      NULL
      ["credit4Producer"]=>
      NULL
      ["memberRole"]=>
      NULL
      ["jobDescription"]=>
      NULL
      ["jobTitle"]=>
      NULL
      ["subscriptionAllowedSubscribers"]=>
      NULL
      ["subscriptionTitle"]=>
      NULL
      ["subscriptionDescription"]=>
      NULL
      ["newsImage"]=>
      NULL
      ["newsArticle"]=>
      NULL
      ["copy"]=>
      NULL
      ["images"]=>
      NULL
      ["newsAllowedViewer"]=>
      NULL
      ["newsExcerpt"]=>
      NULL
      ["subscriptionRegisteredUserGroup"]=>
      NULL
      ["subscriptionPrice"]=>
      NULL
      ["subscriptionsInfo"]=>
      NULL
      ["subscriptionInfoEntry"]=>
      NULL
      ["subscriptionRequirePostcode"]=>
      NULL
      ["acceptedPostcodes"]=>
      NULL
      ["memberViewCount"]=>
      NULL
      ["memberProfileLive"]=>
      NULL
      ["memberProductionJobRole"]=>
      NULL
      ["subscriptionShowJobRole"]=>
      NULL
      ["skillsCategories"]=>
      NULL
      ["equipmentCategories"]=>
      NULL
      ["genresCategories"]=>
      NULL
      ["memberInitialSubscriptionExpiry"]=>
      NULL
      ["rolesCategories"]=>
      NULL
      ["credit1Channel"]=>
      NULL
      ["credit1Company"]=>
      NULL
      ["credit2Channel"]=>
      NULL
      ["credit3Channel"]=>
      NULL
      ["credit4Channel"]=>
      NULL
      ["credit2Company"]=>
      NULL
      ["credit3Company"]=>
      NULL
      ["credit4Company"]=>
      NULL
      ["enupalStripeBasicFields"]=>
      NULL
      ["required"]=>
      NULL
      ["label"]=>
      NULL
      ["fieldHandle"]=>
      NULL
      ["placeholder"]=>
      NULL
      ["initialRows"]=>
      NULL
      ["options"]=>
      NULL
      ["maxValue"]=>
      NULL
      ["minValue"]=>
      NULL
      ["hiddenValue"]=>
      NULL
      ["enupalMultiplePlans"]=>
      NULL
      ["default"]=>
      NULL
      ["setupFee"]=>
      NULL
      ["customLabel"]=>
      NULL
      ["selectPlan"]=>
      NULL
      ["membershipsList"]=>
      NULL
      ["membershipSubscription"]=>
      NULL
      ["membershipEnabled"]=>
      NULL
      ["membershipDescription"]=>
      NULL
      ["membershipTitle"]=>
      NULL
      ["membershipUserGroup"]=>
      NULL
      ["membershipSubscriptionUserGroups"]=>
      NULL
      ["_customFieldValues":"craft\behaviors\ContentBehavior":private]=>
      array(0) {
      }
      ["owner"]=>
      *RECURSION*
    }
  }
  ["where"]=>
  NULL
  ["limit"]=>
  string(1) "1"
  ["offset"]=>
  NULL
  ["indexBy"]=>
  NULL
  ["emulateExecution"]=>
  bool(false)
}
```

**Comment by andrelopez** [comment]
Hi @iamkeir We just released Stripe Payments v2.1.5 with a fix to this issue. Thanks for reporting
